src/app/tutorials/angular2-whatsapp-ionic-cli/3.0.0.ts

```diff
@@ -1,10 +1,10 @@
 const markdownUrlBase = '/manuals/';
 
-export const V2_0_0 = {
+export const V3_0_0 = {
```

Comment:
Really? Why? The new tutorial contains every single step of the old one, it's just a rebased + bug fixed version of the old one with a couple of new chapters. There is not even a single step of the old tutorial which cannot be reproduced with the new one: even the deprecated parts -like `ImagePicker` in chapter 13- have been maintained and rebased prior to being deprecated (chapter 14 replaces chapter 13's `ImagePicker` with `Camera` because it's better maintained). Maintaining "compatibility" with the old tutorial was my precise goal since the beginning.